docs/install.md

```diff
@@ -1,34 +1,59 @@
-# Installation
+# Introduction
 
-To install k0smotron, run the following command:
+This section describes how to install k0smotron on top of an existing k0s
+Kubernetes cluster that allows for creation and management of the k0s
+control plane.
 
-```bash
-kubectl apply -f https://docs.k0smotron.io/{{{ extra.k0smotron_version }}}/install.yaml
-```
+# Hardware requirements
+
+k0smotron does not require any special hardware aside from the one required for
```

Comment:
Could we emphasize here, that k0s requirements here is for workload clusters, not for k0smotron itself?